### DISCUSSION TITLE
help me

### DISCUSSION THREAD

**Discussion by BestWhere** [ggml-org/llama.cpp#9809] [Q&A] [answered]
Hello everyone, I am an undergraduate student in China, I want to learn about large models, can you give me some advice? I really don't know where to start. :(

**Comment by Zapotecatl** [comment, marked as answer]
Hi,

The basis of the LLM are transformers, you could start by understanding the concepts of transformers. I like these series of videos.

[Episode 1](https://www.youtube.com/watch?v=dichIcUZfOw&t)
[Episode 2](https://www.youtube.com/watch?v=mMa2PmYJlCo)
[Episode 3](https://www.youtube.com/watch?v=gJ9kaJsE78k)

Another explanation of transformers.
[transformers](https://www.youtube.com/watch?v=bCz4OMemCcA&t)

Now focused on LLAMA and with code:

[Llama](https://www.youtube.com/watch?v=Mn_9W1nCFLo)
[Llama code](https://www.youtube.com/watch?v=oM4VmoabDAI&t)

I hope it can be useful.

**Reply by BestWhere** [reply]
It would be a great honor to hear from you. I strive to be one percent excellent from you, and I will continue to work hard. I really like the spirit of helping each other on our internet, I will continue to pass it on, thank you!

**Comment by nikhil-arm** [comment]
I really like this explaination : https://jalammar.github.io/illustrated-transformer/

**Reply by BestWhere** [reply]
god bless you！:)
